schedule.html

```diff
@@ -316,7 +441,11 @@ <h4 id="schedule-day-1">
               <dd>
                 Because VR is much better experienced than talked about, we
                 encourage anyone with demos worth sharing to come prepared to
-                show them
+                show them. In order to facilitate identification of what demos
+                will be available from whom, we encourage you to document your
+                demos in <a href=
+                "https://docs.google.com/document/d/1opAAH6CZhxx5F6c9bvISphHF58ZLE7dS4mJKQDQKRDQ/">
```

Comment:
I'm hesitant to put this in the schedule; although that URL is now de facto public, I think I would prefer keeping it to direct messages to participants.